modules/bbp-electrophysiology/src/test/resources/data/bbp/electrophysiology/trace/v0.1.0/all-fields.json

```diff
@@ -0,0 +1,38 @@
+{
+    "@context": [
+        "{{base}}/contexts/nexus/prov/rec-prov-o-20130430/v0.1.0",
+        "{{base}}/contexts/bbp/core/entity/v0.1.0",
+        "{{base}}/contexts/bbp/neurosciencegraph/core/v0.1.0"
+    ],
+
+    "@type": [
+        "prov:Entity",
+        "nsg:Trace",
+        "nsg:StimulationTrace"
+    ],
+    "name": "654363",
+    "providerId": "654363",
+    "description": "Stimulation trace",
+    "channel": "0",
+    "repetition": "2",
+    "citation": "https://doi.org/10.1093/cercor/bhh092",
+    "projectName": "Somatosensory cortex project",
```

Comment:
We should reach an agreement as to what the contents of a projectName will be, for Neobase there are already project names such as "Maria Single Cell". Should the projectName be the one that was given by the experimenter or a more brain region related one?

Reply:
To be handled in an other ticket.